R/get_latest_output_date_index.R

```diff
@@ -12,7 +12,8 @@ get_latest_output_date_index <- function(dir, date) {
   currentfolders <- list.files(dir)
 
   # subset to date
-  date_dirs <- grep(date, currentfolders, value = T)
+  pat <- sprintf("^%s[.]\\d{2}$", date)
```

Comment:
subjective: i always use `[.]` instead of `\.` in regex because it's much easier for me to visually parse.

I picked this up when I was working with lots of URLs and regex, so dots a plenty

Reply:
update: it's `\\.` in R. another reason for me to use `[.]`